application/src/TeamData.ts

```diff
@@ -1,103 +1,130 @@
+import { Color } from "./utils/Color";
+import { Match } from "./Utils";
+import { SectionData } from "./strategy/charts/PieChart";
+import Percent from "./utils/Percent";
+import { FieldObject, mapButtons } from "./scouter/inputtypes/MapInput";
+
+interface Comment {
+  body: string;
+  qual: number;
+}
+
+interface MapFields {
+  SpeakerScore: number;
+  SpeakerMiss: number;
+
+  PassSuccessful: number;
+  PassUnSuccessful: number;
+}
+
+type FullMatch = Match & MapFields;
+
 export class TeamData {
-  public readonly matches: Record<string, Record<string, string>>;
-  [key: string]: any;
+  private matches: FullMatch[];
 
-  public static readonly matchName = "Qual";
-  public static readonly mapName = "CRESCENDO";
+  static exampleData: TeamData = new TeamData([
+    {
+      ScouterName: "Yoni",
+      Qual: 1,
+      TeamNumber: 4590,
+      GameSide: "Blue",
+      StartingPosition: "Amp Side",
+      SpeakerAutoScore: 3,
+      SpeakerAutoMiss: 4,
+      MapPoints: [
+        { x: 100, y: 300, successfulness: true, pressedButton: mapButtons[0] },
+      ],
+      AmpScore: 2,
+      AmpMiss: 1,
+      Climb: "Park",
+      Trap: "Scored",
+      Comment: "lol",
```

Comment:
![image](https://github.com/user-attachments/assets/30d81b9c-6874-4c11-a91e-7cbae54e96da)
(just kidding, you can keep this)